kvm.rb

```diff
@@ -2,8 +2,10 @@
 
 class Kvm < Formula
   homepage "https://www.github.com/aspnet/Home"
-  version "1.0.0-beta1"
-  url "https://github.com/aspnet/Home.git", :branch => 'master'
+  version "1.0.0-beta3"
+  url "https://github.com/aspnet/Home.git", :tag => 'v1.0.0-beta3'
+
+  head "https://github.com/aspnet/Home.git", :branch => 'dev'
```

Comment:
can we have long term solution where we don't need to keep updating this every release ?